src/Orleans.CodeGenerator/Model/WellKnownCodecDescription.cs

```diff
@@ -2,20 +2,19 @@
 
 namespace Orleans.CodeGenerator
 {
-    internal class WellKnownCodecDescription : ICodecDescription
+    internal sealed class WellKnownCodecDescription
     {
         public WellKnownCodecDescription(ITypeSymbol underlyingType, INamedTypeSymbol codecType)
         {
             UnderlyingType = underlyingType;
             CodecType = codecType;
         }
 
-        public ITypeSymbol UnderlyingType { get; }
-
-        public INamedTypeSymbol CodecType { get; }
+        public readonly ITypeSymbol UnderlyingType;
+        public readonly INamedTypeSymbol CodecType;
     }
 
-    internal class WellKnownCopierDescription : ICopierDescription
+    internal sealed class WellKnownCopierDescription : ICopierDescription
```

Comment:
I suppose we could also delete `ICopierDescription`

Reply:
I briefly looked into it, but it wasn't as clear cut as the ICodecDescription case.